Assets/Sprites/Health bar 4.png.meta

```diff
@@ -0,0 +1,122 @@
+fileFormatVersion: 2
```

Comment:
`Health bar 4.png` could be renamed to `Health Bar Border.png` for the file name to be more clear, but it doesn't really matter much.